.github/workflows/offline-docs.yml

```
@@ -0,0 +1,63 @@
+name: Update Offline Docs
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
```

In order to respond to pushes on `theos.dev`, we would need a repository dispatch `on` trigger.

We can re-use the organization-wide trigger for this. I think that ideally `theos.dev` would have it's own trigger, but I don't think it's a big enough deal to setup a separate handler for.

```suggestion
  repository_dispatch:
    types: [org_push]
```